data/yam/agama/auto/alp_bedrock.sh

```diff
@@ -1,7 +1,7 @@
 set -ex
```

Comment:
might help to call them: `default_alp_bedrock` and `default_alp_micro` as you did for tw, to distinguish them from further work.